This patch disables building of the (effectively empty)
dl-iterate-phdr.c source file.  Symbols generated by annobin confuse
the run-check-namespace test because nm -g prints hidden weak symbols
in DSOs (they are considered external):

  ERROR: Extraneous symbols:
000000000000de51 W dl_iterate_phdr.c.a8d8d212
  ERROR: Extraneous symbols:
00000000000120b0 W dl_iterate_phdr.c.a8d8d212

This patch is downstream-specific due to annobin.

diff --git a/src/Makefile.am b/src/Makefile.am
index 2b5b02959e99eb8f..e5ff21515c36d30f 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -147,7 +147,7 @@ libunwind_la_SOURCES_local =						\
 	$(libunwind_la_SOURCES_local_unwind)
 
 noinst_HEADERS += os-linux.h
-libunwind_la_SOURCES_os_linux = os-linux.c dl-iterate-phdr.c
+libunwind_la_SOURCES_os_linux = os-linux.c
 
 libunwind_la_SOURCES_os_hpux = os-hpux.c
 
